src/phoenix/trace/trace_dataset.py

```diff
@@ -98,15 +98,19 @@ class TraceDataset:
     """
 
     name: str
+    """
+    A human readable name for the dataset.
+    """
     dataframe: pd.DataFrame
     evaluations: List[Evaluations] = []
-    _id: UUID = uuid4()
+    _id: UUID
```

Comment:
this was making all trace datasets have the same ID I believe

Reply:
Good catch.